jetson/percep/artag_detector.cpp

```diff
@@ -20,7 +20,7 @@ TagDetector::TagDetector() {  //initializes detector object with pre-generated d
 
     cv::FileStorage fsr("jetson/percep/alvar_dict.yml", cv::FileStorage::READ);
     if (!fsr.isOpened()) {  //throw error if dictionary file does not exist
-        std::cerr << "ERR: \"alvar_dict.yml\" does not exist! Create it before running main\n";
+        std::cout << "ERR: \"alvar_dict.yml\" does not exist! Create it before running main\n";
```

Comment:
Sorry to keep changing things up on you guys, but for the sake of consistency we are going to use cerr for errors that shouldn't happen and cout for normal output

Reply:
Fixed, all error outputs changed to cerr